Dockerfile

```diff
@@ -11,11 +11,21 @@ RUN cd \
   && curl -sS https://getcomposer.org/installer | php \
   && ln -s /root/composer.phar /usr/local/bin/composer
 
+
+FROM common as dev
 ADD ./ /code
+WORKDIR /code
+
+RUN composer install \
+    --prefer-dist \
+    --no-interaction \
+    --dev
```

Comment:
imho je opačně - by default je to všechno a musíš do produkce dopsat  [`--no-dev`](https://getcomposer.org/doc/03-cli.md#install-i)

Reply:
změnil jsem ten dockerfile úplně